docs/data.md

```diff
@@ -104,6 +104,8 @@ E.g. for PyPI vulnerabilities:
 gsutil cp gs://osv-vulnerabilities/PyPI/all.zip .
 ```
 
+Some ecosystems contain a `:` separator in the name (e.g. `Alpine:v3.17`), for these ecosystems, the data dump will always contain the same ecosystem without `:.*` suffix (e.g. `Alpine`), which will contain all the advisories of the ecosystem with the same prefix (e.g. All `Alpine:.*`).
```

Comment:
nit: this sentence is quite long, so I'd suggest breaking it up a little bit + make some other small clarifications. 

```suggestion
Some ecosystems contain a `:` separator in the name (e.g. `Alpine:v3.17`). For these ecosystems, the data dump will always contain an ecosystem directory without the `:.*` suffix (e.g. `Alpine`). This will contain all the advisories of the ecosystem with the same prefix (e.g. All `Alpine:.*`).
```